src/odemis/driver/test/actuator_test.py

```diff
@@ -711,5 +713,396 @@ def test_fail_sensor(self):
             f.result()  # should raise an error
 
 
+class TestCombinedFixedPostionActuator(unittest.TestCase):
```

Comment:
There is a `i` missing in the class name.

Reply:
Done, also changed in tearDown function